source/fa/1.0.0/index.html.haml

```diff
@@ -0,0 +1,250 @@
+---
+description: تاریخچهٔ تغییرات را نگه دارید
+title: تاریخچهٔ تغییرات را نگه دارید
+language: fa
+version: 1.0.0
+---
+<link type="text/css" rel="stylesheet" href="https://cdn.rawgit.com/rastikerdar/vazir-font/v19.0.0/dist/font-face.css">
```

Comment:
I'd prefer if we didn't hotlink random CDNs since this could change or break in the future, is there an alternative for us to add this CSS to the repo instead? 

More importantly RawGit.com is shutting down, so this won't work for long: 
<img width="814" alt="image" src="https://user-images.githubusercontent.com/65950/224201448-9ddc9b96-5829-454d-9aea-5e78703829b4.png">

Reply:
You can use this popular Persian fonts:
https://github.com/rastikerdar/vazirmatn
https://fonts.google.com/specimen/Vazirmatn